nodejs/image.bzl

```diff
@@ -59,6 +61,14 @@ DEFAULT_BASE = select({
     "//conditions:default": "@nodejs_debug_image_base//image",
 })
 
+load("@build_bazel_rules_nodejs//:defs.bzl", "SourcesInfo")
```

Comment:
can you alias this with a name that is more descriptive of this being specific a nodejs definition (could you also please comment on what it does in _app_runfiles below?)